7/zts/vendor/karibbu/Dockerfile

```diff
@@ -51,6 +52,7 @@ opcache.fast_shutdown=1\n\
     && apk add --no-cache icu \
     && docker-php-source delete \
     && apk add --no-cache make \
+    && apk add --no-cache coreutils \
```

Comment:
I would prefer you move this line here: https://github.com/rezzza/docker-php/blob/635696fcd3adc3db4001938d661c6c51889773a6/7/zts/vendor/karibbu/Dockerfile#L21